### PR TITLE
Editorial pass

### PR DESCRIPTION
**Pull request by mcherem**

### CODE REVIEW

File: edifact-edi/2.2/modules/ROOT/pages/edifact-edi-connector-config-topics.adoc [hunk 1]
```diff
@@ -68,38 +70,38 @@ These control the parser operation and the types of error conditions that reject
...
+|enforceLengthLimits="true" |Enforce the minimum and maximum lengths for receive values.
+|enforceValueRepeats="true" |Enforce the repetition count limits for receive values.
+|enforceReadCharacters="true" |Enforce valid characters for a Read. Only applies when using syntax levels `UNOA` and `UNOB`.
+|allowUnknownSegments="false" |Allow unknown segments in a message.
+|enforceSegmentOrder="true" |Enforce the segment order in a message.
+|allowUnusedSegments="false" |Allow segments marked as `Unused` in a message.
+|enforceSegmentRepeats="true" |Enforce segment repetition count limits in a message.
+|receiveEncodingOverride="false" |Java character encoding used for Receive messages. This overrides receive UNB.
+|requireUniqueInterchanges="true" |Require unique Interchange Control References (such as UNB.5). The default behavior records the interchange numbers previously processed and rejects duplicate interchange numbers from the same partner. It is derived from the interchange Sender and Receiver identification. Setting to `false` enables received interchange processing to continue and relies on the application flow to track the numbers and handle them appropriately.
```
Comment (kevintroller) [inline edifact-edi/2.2/modules/ROOT/pages/edifact-edi-connector-config-topics.adoc]:
```suggestion
|requireUniqueInterchanges="true" |Require unique Interchange Control References (such as UNB.5). The default behavior records the interchange numbers previously processed and rejects duplicate interchange numbers from the same partner. It is derived from the interchange Sender and Receiver identification. Setting to `false` enables received interchange processing to continue and relies on the application flow to track the numbers and manage them appropriately.
```

Comment (kevintroller) [reply edifact-edi/2.2/modules/ROOT/pages/edifact-edi-connector-config-topics.adoc]: Kim many times edited _handle_ to **manage**. Feel free to take it or leave it

File: edifact-edi/2.2/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -16,7 +16,7 @@ To use this connector, you must be familiar with:
 * Elements and global elements in a Mule flow
 * Creating a Mule app using Anypoint Studio (Studio)
 
-To use EDIFACT Connector, you must have:
+To use EDIFACT connector, you must have:
```
Comment (kevintroller) [inline edifact-edi/2.2/modules/ROOT/pages/index.adoc]:
```suggestion
To use EDIFACT EDI Connector, you must have:
```

**Review by kevintroller — DISMISSED** [review]
LGTM, minor comments added

File: edifact-edi/2.2/modules/ROOT/pages/edifact-edi-connector-config-topics.adoc (hunk 1 above)
Comment (mcherem) [reply edifact-edi/2.2/modules/ROOT/pages/edifact-edi-connector-config-topics.adoc]: Thanks!

File: edifact-edi/2.2/modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (mcherem) [reply edifact-edi/2.2/modules/ROOT/pages/index.adoc]: Fixed, thanks!

**Review by kevintroller — APPROVED** [review]

**Pull request merged**